### ISSUE TITLE
Create Automation based off of Member engagement

### ISSUE THREAD

**Issue by stephenhutson** [Simplero/Roadmap#178]
We would like to create automation based on how active a contact has been, meaning how many pages within a membership site they have views, courses completed, resources downloaded, number of Forum comments, etc. The idea is to create an email sequence that sends only if the contact has been inactive for 7 days, or send an email automatically once they complete 5 courses, etc.

**Comment by calvincorreli** [comment]
This would be fantastic. I'm trying to think how it would be implemented.

We could have triggers on the various events (view page, login to membership site, download a resource, etc.), and then with conditions that let you check how many resources have been downloaded / pages viewed, etc. These would probably be expressed in liquid expression syntax, just to make it consistent.

Sound about right to your ears?

**Comment by stephenhutson** [comment]
Yes that sounds right! Except I have no idea what liquid expression syntax is, so I'll leave that to you :)

For the triggers, here's an initial list:

- date of last login
- view page
- download a resource
- start a course
- complete a course
- number of courses completed
- number of forum comments/posts
- number of other comments (courses, blogs, etc) 
- completed their profile

**Comment by AndreaSchroeder77** [comment]
Yes this would be great!
